src/TStringUtils.h

```diff
@@ -54,6 +54,29 @@ class TStringUtils
             func(ptr);
         }
     }
+
+    static void listAddItem(QString &list, const QString &item)
+    {
+        if (!list.isEmpty()) {
+            list.append("|");
```

Comment:
Avoid raw string literals as we should be wrapping them in `tr(`...`)` or `qsl(`...`)` depending on if they are to be translated or not - in this case we are only dealing with a single character so we can avoid any confusion by using a `QChar`:
```suggestion
            list.append(QLatin1Char('|'));
```